client/src/components/ImportPanel/index.tsx

```diff
@@ -93,8 +94,14 @@ const ImportPanel: React.FC<ImportPanelProps> = (props) => {
     axios
       .get(url)
       .then((res) => {
-        alert("Success");
-        setData(res.data);
+        console.log(res.data);
+        console.log("Creating Archipelago");
+        console.time("Archipelago");
+        const javaArchipelago = new JavaArchipelago(res.data);
+        console.timeEnd("Archipelago");
+        console.log("Done!");
+        alert("Success.");
+        setData(javaArchipelago);
```

Comment:
I think those `console.log` statements should be removed before merging this PR.

Reply:
Ok, I deleted them